product-configuration-extension/shopify.ui.extension.toml.liquid

```diff
@@ -0,0 +1,16 @@
+name = "{{ name }}"
+type = "ui_extension"
+
+api_version = "unstable"
```

Comment:
@vividviolet Is there a specific API version we should be using?

Reply:
Yes, `2023-07` should be the target